plugins/wazuh-core/server/initialization/server-api.test.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  PLUGIN_APP_NAME,
+  PLUGIN_PLATFORM_WAZUH_DOCUMENTATION_URL_PATH_TROUBLESHOOTING,
+} from '../../common/constants';
```

Comment:
You shouldn't use any constats in your tests, you should use hardcoded values instead.

Reply:
That constants are used in the source code, defining a new constant for the test implies the definition is duplicated. If the source code and the test are using the same variable, changing the variable value, should not cause the test fails, for this reason these constant are used.